ee/clickhouse/views/experiment_holdouts.py

```diff
@@ -50,6 +50,16 @@ def _get_filters_with_holdout_id(self, id: int, filters: list) -> list:
             )
         return updated_filters
 
+    def validate_filters(self, filters):
+        for filter in filters:
+            rollout_percentage = filter.get("rollout_percentage")
+            if rollout_percentage is None:
+                raise serializers.ValidationError("Rollout percentage must be present.")
+            if rollout_percentage < 0 or rollout_percentage > 100:
```

Comment:
Note that this could be simplified to: `if not (0 <= rollout_percentage <= 100):`

Notice the variable only needs to be used once. But it's a stylistic preference and not a blocker here.